posthog/client.py

```diff
@@ -724,7 +724,7 @@ def get_feature_flag_payload(
                 person_properties=person_properties,
                 group_properties=group_properties,
                 send_feature_flag_events=send_feature_flag_events,
-                only_evaluate_locally=True,
+                only_evaluate_locally=only_evaluate_locally,
```

Comment:
the bug in this case was that we were not evaluating the feature flag payload if we couldn't compute it locally, which shouldn't happen in the event where we still want to use remote evaluation at some point – ideally we won't fall back, but we should _sometimes_, and making this payload explicitly true was preventing us from falling back always.